cli/smartcontract/generate_test.go

```diff
@@ -325,6 +325,8 @@ func (c *ContractReader) Get() (*big.Int, error) {
 }
 
 func TestGenerateRPCBindings(t *testing.T) {
+	const rewriteExpectedOutputs = false
```

Comment:
Make it global? You likely want to update all of them at once, although I've done it manually previously (one liner).

Reply:
Fixed.